src/app/oauth2/middleware.ts

```diff
@@ -41,10 +41,11 @@ export const handleOAuthJourneyAction: RequestHandler = async (
   next,
 ) => {
   try {
-    checkForIpvAndOauthSessionId(req, res);
+    if (!req.session.ipvSessionId) {
```

Comment:
For the cross-browser scenario we need to allow just clientOauthSessionId, but maybe this route is unaffected?

Reply:
Yeah, I don't think that will apply here - this is only ever called on an authorize call, immediately after `setIpvSessionId`.

TBH we could probably just combine these two 'middlewares' into a single request handler. Separating the two like this doesn't seem like a useful abstraction.